### ISSUE TITLE
Md-datepicker input does not have a valid label

### ISSUE THREAD

**Issue by manish2535** [angular/material#9398]
**Actual Behavior**:
 -  `What is the issue?` Md-datepicker input does not have a valid label.
 -  `What is the expected behavior?` Md-datepicker input should have a label for accessibility purpose

`<input class="md-datepicker-input" aria-haspopup="true" ng-focus="ctrl.setFocused(true)" ng-blur="ctrl.setFocused(false)" readonly="readonly" size="12" placeholder="Trip Date">`

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:`
 -  `Details :` You can replicate the problem on the md-datepicker demo page https://material.angularjs.org/latest/demo/datepicker

**Angular Versions**: *
 -  `Angular Version:` v1.4.7
 -  `Angular Material Version:` 1.1.0-rc.5

**Additional Information**:
 -  `Browser Type:` All browsers
 -  `Browser Version: *`
 -  `OS:` Windows 7
 -  `Stack Traces:`

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by crisbeto** [comment]
Are you talking about the `<label>` element or an `aria-label`?

**Comment by manish2535** [comment]
I am taking about Labels in general. Any input element should have either a <label> element or an aria-label associated with it. This input field does not have either.

**Comment by ThomasBurleson** [comment]
@crisbeto - `aria-label`

**Comment by crisbeto** [comment]
The `aria-label` was sorted out recently with https://github.com/angular/material/pull/9364.

**Comment by manish2535** [comment]
Thanks @crisbeto and @ThomasBurleson
